README.md

```diff
@@ -23,7 +23,17 @@ Currently, the following encoding tokens are supported, using the first acceptab
 4. `*` (no preference — `fastify-compress` will use `gzip`)
 5. `identity` (no compression)
 
-If the `accept-encoding` header is missing or contains no supported encodings, it will not compress the payload.
+If an unsupported encoding is received or if the `'accept-encoding'` header is missing, it will not compress the payload. If an unsupported encoding is recieved, a custom error response can be sent in place of the uncompressed payload by setting the `onUnsupportedEncoding(encoding, reply)` option to be a function that can modify the reply and return a `string | Buffer | Stream | Error` payload.
```

Comment:
```suggestion
If an unsupported encoding is received or if the `'accept-encoding'` header is missing, it will not compress the payload. If an unsupported encoding is received, a custom error response can be sent in place of the uncompressed payload by setting the `onUnsupportedEncoding(encoding, reply)` option to be a function that can modify the reply and return a `string | Buffer | Stream | Error` payload.
```